### PR TITLE
Mikem 97

### PR DESCRIPTION
**Pull request by mmastand**
adds an AUC function to model_eval

### CODE REVIEW

**Comment by levithatcher** [comment]
Hey Mike! Let me know if you have any questions about my comments. Thanks for writing this up! You should be able to merge when needed.

File: healthcareai/common/model_eval.py [hunk 1]
```diff
@@ -261,5 +263,75 @@ def calculate_rfmtry(number_of_columns, type):
...
+    aucType (str) : either 'SS' for ROC curve or 'PR' for precision recall curve.
+    plotFlg (bol) : True will return plots.
+
+    Returns
+    -------
+    Nothing. Simply prints feature importance list to console.
+    """
+    # Error check for uneven length predictions and labels
+    if len(predictions) != len(labels):
+        sys.exit('Data vectors are not equal length!')
```
Comment (mmastand) [inline healthcareai/common/model_eval.py]: I changed this to raise an exception

File: healthcareai/common/model_eval.py [hunk 2]
```diff
@@ -261,5 +263,75 @@ def calculate_rfmtry(number_of_columns, type):
...
+            plt.xlim([0.0, 1.0])
+            plt.ylim([0.0, 1.05])
+            plt.xlabel('False Positive Rate')
+            plt.ylabel('True Positive Rate')
+            plt.title('Receiver operating characteristic curve')
+            plt.legend(loc="lower right")
+            plt.show()
+
+    # Compute PR curve and PR area
+    if aucType == 'PR':
```
Comment (mmastand) [inline healthcareai/common/model_eval.py]: Of course! no problem.

File: healthcareai/common/model_eval.py [hunk 3]
```diff
@@ -261,5 +263,75 @@ def calculate_rfmtry(number_of_columns, type):
...
+        # Compute Precision-Recall and plot curve
+        precision, recall, _ = precision_recall_curve(labels, predictions)
+        average_precision = average_precision_score(labels, predictions)
+        print('Area under PR curve (AU_PR): %0.2f' % average_precision)
+
+        # plot PR curve
+        if plotFlg == True:
```
Comment (mmastand) [inline healthcareai/common/model_eval.py]: switched them.

**Pull request merged**